src/iso_alloc.c

```diff
@@ -67,6 +67,16 @@ INTERNAL_HIDDEN void iso_alloc_initialize_global_root(void) {
      * result in a soft page fault */
     MLOCK(&_root, sizeof(iso_alloc_root));
 
+#if ARM_MTE
+    if(iso_is_mte_supported() == false) {
```

Comment:
Random driveby: I would allow code that uses an allocator built with MTE support to also run on devices that don't support it. That simplifies deployment  a lot.

Reply:
Random drive by appreciated!